sphere-models/src/main/java/io/sphere/sdk/types/Custom.java

```diff
@@ -0,0 +1,5 @@
+package io.sphere.sdk.types;
+
+public interface Custom {
```

Comment:
Still not convinced about the naming. Wouldn't it be better `Customizable` or `CustomizableFields`?